AppCenterDataStorage/AppCenterDataStorage/Internal/Client/MSCosmosDb.m

```diff
@@ -133,13 +134,31 @@ + (void)performCosmosDbAsyncOperationWithHttpClient:(id<MSHttpClientProtocol>)ht
                                         tokenResult:(MSTokenResult *)tokenResult
                                          documentId:(NSString *)documentId
                                          httpMethod:(NSString *)httpMethod
-                                               body:(NSData *_Nullable)body
+                                           document:(id<MSSerializableDocument> _Nullable)document
                                   additionalHeaders:(NSDictionary *_Nullable)additionalHeaders
+                                  additionalUrlPath:(NSString *_Nullable)additionalUrlPath
                                   completionHandler:(MSHttpRequestCompletionHandler)completionHandler {
+  NSData *body = nil;
+  if (document) {
+    // Serialize document
+    NSError *serializationError;
```

Comment:
I don't see new unit tests for those new cases.

Reply:
Looks like this is moved from somewhere and that's why there is no new unit tests.
I think the problem is that old code hasn't been cleaned.